### PR TITLE
Sr orgnr safeguard

### PR DESCRIPTION
**Pull request by simen-rekkedal**
## Related Issue(s)
- #{issue number}

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/Authentication/appsettings.Development.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "SBLRedirectEndpoint": "https://at22.altinn.cloud/ui/authentication/",
     "SBLLogoutEndpoint": "https://at22.altinn.cloud/ui/authentication/logout",
     "PlatformEndpoint": "https://platform.at22.altinn.cloud/",
+    "ApiAccessManagementEndpoint": "http://platform.at22.altinn.cloud/accessmanagement/api/v1/",
```
Comment (acn-dgopa) [inline src/Authentication/appsettings.Development.json:12]: it should be https

File: src/Authentication/appsettings.Development.json [hunk 2]
```diff
@@ -65,5 +66,8 @@
     "AuthenticationDbUserConnectionString": "Host=localhost;Port=5432;Username=auth_authentication;Password=postgres;Database=authentication",
     "AuthenticationDbAdminPassword": "postgresAdmin",
     "AuthenticationDbPassword": "postgres"
+  },
+  "AccessManagementSettings": {
+    "ApiAccessManagementEndpoint": "http://platform.at22.altinn.cloud/accessmanagement/api/v1/"
```
Comment (acn-dgopa) [inline src/Authentication/appsettings.Development.json:71]: https

File: src/Persistance/RepositoryImplementations/SystemUserRepository.cs [hunk 3]
```diff
@@ -52,8 +52,7 @@ await command.ExecuteEnumerableAsync()
         }
         catch (Exception ex)
         {
-            _logger.LogError(ex, "Authentication // SystemRegisterRepository // SetDeleteSystemUserById // Exception");
-            throw;
+            _logger.LogError(ex, "Authentication // SystemRegisterRepository // SetDeleteSystemUserById // Exception");
```
Comment (acn-dgopa) [inline src/Persistance/RepositoryImplementations/SystemUserRepository.cs:55]: systemuserrepostiory

**Review by acn-dgopa — APPROVED** [review]

File: src/Authentication/appsettings.Development.json (hunk 1 above)
Comment (simen-rekkedal) [reply src/Authentication/appsettings.Development.json:12]: ok, fixed

File: src/Authentication/appsettings.Development.json (hunk 2 above)
Comment (simen-rekkedal) [reply src/Authentication/appsettings.Development.json:71]: ok, fixed

File: src/Persistance/RepositoryImplementations/SystemUserRepository.cs (hunk 3 above)
Comment (simen-rekkedal) [reply src/Persistance/RepositoryImplementations/SystemUserRepository.cs]: ok, fixed

**Pull request merged**